osf/models/quickfiles.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals
+
+import logging
+
+from osf.models.node import AbstractNode, AbstractNodeManager
+from website.exceptions import NodeStateError
+
+
+logger = logging.getLogger(__name__)
+
+
+class QuickFilesManager(AbstractNodeManager):
+
+    def create_for_user(self, user):
+        possessive_title = get_quickfiles_project_title(user)
+
+        quickfiles, created = QuickFiles.objects.get_or_create(
+            title=possessive_title,
+            creator=user
+        )
+
+        if not created:
+            raise NodeStateError('Users may only have one quickfiles project')
+
+        quickfiles.add_addon('osfstorage', auth=None, log=False)
+
+        return quickfiles
+
+    def get_for_user(self, user):
+        return QuickFiles.objects.get(creator=user)
+
+
+class QuickFiles(AbstractNode):
```

Comment:
Sorry for the confusion! The comment you linked to still stands--`node.is_quickfiles` is correct. The model name is what I am proposing to change--properties and methods can stay the same.

That said, I don't feel strongly about changing it. I'll let you have final say on whether or not it's worth it.